starboard/CHANGELOG.md

```diff
@@ -406,726 +412,3 @@ to improve the performance. The caches store the results of previous queries and
 reuse them for repeated queries. Note that if caches are enabled, the platform
 need to clear the caches if there's any codec or audio/video output capability
 change.
-
-## Version 13
```

Comment:
@kaidokert for your opinion—seems weird to me to remove entries from the changelog entirely, even if the versions are deprecated, though I see it already only goes back to SB10

Reply:
hi @kaidokert , @lamdbui , would yo like to keep the old versions of change logs? I removed them because old versions are removed in existing CHANGELOG.md file.

Reply:
I looked at it at the start and had the same question. But then, it was previously trimmed already.

In favor of keeping it - it's nice sometimes to just quickly search to see how far back some things go. But i've noticed it's not always reliable either as things were missed from changelog. Most decently managed projects also keep changelog almost forever, unless its so long that old entries get moved into a separate file. We are not there yet.

In favor of trimming it - it was previously trimmed back to supported versions and we don't support those versions anymore. The old copy of it is always available in old branches.

I'd maybe lean on the side of leaving it included - if uncertain, keep the status quo.

Reply:
okay.  the change logs of old versions have been restored.